### ISSUE TITLE
Request: Notification pop up with custom sound?

### ISSUE THREAD

**Issue by tgp1994** [Belphemur/SoundSwitch#73]
I just switched over to this app from the old one (didn't realize development has changed!) and it's looking great so far.

I took advantage of the new option to specify your own sound, which is great. However, it appears that this disables the visual notification pop up as well. Sometimes, my hotkey doesn't get recognized or go through so I'm not sure if playback has actually switched to my other device (which may be turned down too low) or the hotkey was actually missed. This usually leads to me spamming the hotkey until I know for sure that it has changed :)

TL;DR: It would be great to have an option (check box?) that toggles whether or not a visual notification accompanies the Windows sound/custom sound that we choose.

**Comment by Belphemur** [comment]
I was thinking of doing this in the  beginning but couldn't find any code example on how to customize toast notification when not doing a WPF application.

Well anyway, I found this https://www.whitebyte.info/programming/c/how-to-make-a-notification-from-a-c-desktop-application-in-windows-10
and this
https://msdn.microsoft.com/en-us/library/windows/apps/br230849.aspx

As soon I have finish with the autoupdate, I'll work on this feature.

**Comment by tgp1994** [comment]
Great, thank you!

**Comment by Belphemur** [comment]
This wasn't as easy as I expected it, but now it's available in the last beta
